app/src/main/java/ayds/winchester/songinfo/home/view/DateDescription/DatePrecisionText.kt

```diff
@@ -0,0 +1,60 @@
+import ayds.winchester.songinfo.home.model.entities.DatePrecision
+
+interface DateText{
+    fun getTextDate(date: String, precision: DatePrecision): String
+}
+
+object DateTextImpl : DateText{
```

Comment:
En general vamos a evitar los singletons en este tipo de clases, podríamos querer otras implementaciones. Además, debería ser internal
```suggestion
internal class DateTextImpl : DateText{
```